app/assets/stylesheets/components/login_page.scss

```diff
@@ -171,4 +171,17 @@
       font-family: FontAwesome;
     }
   }
+
+  .tou-caution {
```

Comment:
Since we're using SASS, the styles can also be written as

```sass
.tou-caution {
  margin: 0 0 24px 0;

  a  {
    color: $grey;
    font-size: 0.9em;

    &:hover {
      color: $text-primary;
    }
  }
}
```